raiden/tests/integration/test_blockchainservice.py

```diff
@@ -257,8 +259,13 @@ def test_blockchain(
     assert channel_manager_address == event['channel_manager_address'].decode('hex')
     assert token_proxy.address == event['asset_address'].decode('hex')
 
+    contract_data = solidity_get_contract_data(
+        registry_contracts,
+        get_contract_path('ChannelManagerContract.sol'),
+        'ChannelManagerContract'
+    )
     channel_manager_proxy = jsonrpc_client.new_contract_proxy(
-        registry_contracts['ChannelManagerContract']['abi'],
+        contract_data['abi'],
```

Comment:
Okay, so never mind my comment then, I had the impression that `solidity_get_contract_data` called the compiler again.